src/Tribe/Customizer/Global_Elements.php

```diff
@@ -129,7 +129,10 @@ public function setup() {
 		$description         = $views_v2_is_enabled ? '' : esc_html__( 'Options selected here will override what was selected in the "General Theme" section.', 'the-events-calendar' );
 
 		$this->defaults = [
-			'link_color' => '#141827',
+			'link_color'               => '#141827',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 12 space(s) between "'link_color'" and double arrow, but found 15.

<sub>hash: dfcddde436210a7564e5eab2968dc8dc</sub>